build/config/config-downgrade.php

```diff
@@ -64,6 +63,7 @@
                 \Symplify\RuleDocGenerator\Contract\Category\CategoryInfererInterface::class,
                 \PhpParser\PrettyPrinterAbstract::class,
                 \Helmich\TypoScriptParser\Parser\Traverser\Visitor::class,
+                \Symplify\SymplifyKernel\Contract\LightKernelInterface::class,
```

Comment:
somehow overlapped with `RectorKernel`, so register to typed safe for DowngradeParameterTypeWideningRector